lib/gui/components/safe-webview.js

```diff
@@ -46,6 +46,22 @@ const VERSION_PARAM = 'etcher-version';
 const ELECTRON_SESSION = 'persist:success-banner';
 
 /**
+ * @summary API version search parameter key
+ * @constant
+ * @private
+ * @type {String}
+ */
+const API_VERSION_PARAM = 'api-version';
+
+/**
+ * @summary Webview API version
+ * @constant
+ * @private
+ * @type {String}
+ */
+const API_VERSION = 1;
```

Comment:
Should we start off at 2, and treat `API_VERSION` 1 as the version from before the `api-version` key was added?